requirements.txt

```diff
@@ -153,8 +151,6 @@ idna==2.8
     #   urllib3
 importlib-metadata==1.6.0
     # via -r requirements.in
-importlib-resources==5.7.1
```

Comment:
What about this?

Reply:
This is now not removed rather looks like it's been updated. I assume this is safe if CI passes